media/css/firefox/firstrun/firstrun-quantum.scss

```diff
@@ -11,197 +11,122 @@ $image-path: '/media/protocol/img';
-    }
+.fxa-signin {
+    font-size: 65%;
+    margin-top: 1em;
+}
 
-    @media #{$mq-medium} {
-        @include bidi(((text-align, left, right),));
-    }
+.fxa-signin, .agreement {
+    color: $color-gray-70;
 }
 
+.page-content {
+    margin-bottom: 3em;
+    text-align: center;
+}
 
-//* -------------------------------------------------------------------------- */
-// Animations & transitions
-
-.js {
-    #main-header h1,
-    #main-header .content,
-    #main-header a {
-        opacity: 0;
-        transform: translateY(-5px);
-    }
+/* -------------------------------------------------------------------------- */
+// Emphasis Box style, this should go away when Protocal PR #441 is landed. Issue #385
 
-    #fxaccounts-wrapper {
-        opacity: 0;
-        transform: scale(.8);
-    }
+.mzp-c-box-emphasis {
+    background: $color-white;
+    border-radius: 0.2em;
+    box-shadow: 0 0 10px 0 $color-gray-30;
+    text-align: center;
 
-    // transition in
-    #scene[data-content=true] {
-        #fxaccounts-wrapper {
-            opacity: 1;
-            transform: scale(1);
-            transition: opacity 1s 1s, transform 1s 1s;
-        }
-
-        #main-header {
-            h1,
-            .content,
-            a {
-                opacity: 1;
-                transform: translateY(0);
-                transition: transform .5s, opacity .8s;
-            }
-
-            h1 {
-                transition-delay: 0.5s;
-            }
-
-            .content {
-                transition-delay: 0.7s;
-            }
-
-            a {
-                transition-delay: 1.2s;
-            }
-        }
+    .mzp-c-box-emphasis-content {
+        padding: 1em;
     }
 }
 
+@media #{$mq-sm} {
 
-//* -------------------------------------------------------------------------- */
-// FxA form
+    div.mzp-c-hero-image {
+       height: 100%;
+       vertical-align: top;
+    }
 
-#fxaccounts-wrapper {
-    @include light-links;
-    color: $color-white;
+    section .mzp-l-content, div.mzp-c-hero-image {
+        max-width: 600px;
+        position: relative;
+    }
 
-    .fxa-signin {
-        @include text-body-md;
-        margin-top: $spacing-md;
-        text-align: center;
+    .mzp-c-hero-title {
+        font-size: 2.8em;
     }
-}
 
-.fxa-email-form {
-    color: $color-white;
-    padding: 0 $spacing-md;
+    .mzp-c-box-emphasis-content {
+        padding: 2em;
+    }
 
     .fxa-email-form-intro {
-        @include text-body-lg;
+        font-size: 2em;
     }
 
-    .agreement {
-        @include text-body-xs;
-    }
+}
 
-    label {
-        @include visually-hidden;
-    }
+@media #{$mq-lg} {
 
-    input[type='email'] {
-        @include border-box;
-        @include text-body-md;
-        border-radius: 2px;
-        padding: ($spacing-md - 2px) ($spacing-xl - 2px); //2px extra padding removed to compensate for 2px border.
-        width: 100%;
+    div.mzp-c-hero-image {
+       margin-top: 8em;
```

Comment:
This margin is still in effect when the image first stacks under the box, somewhere between desktop and tablet sizes. Probably just need to find the right breakpoint to change it. Also it should probably be `$layout-xl` or `$layout-2xl` 

![image](https://user-images.githubusercontent.com/205591/62256699-31f46c80-b3b7-11e9-93e1-645028582780.png)